setup.cfg

```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts =-x -v -rw --durations=10 --cov=spyder --cov-report=term-missing
```

Comment:
These options are for:
```
-x exit at the first error
-v verbose
-rw display internal (poytest) warnings
--duration=10 show the 10 most slowest test
--cov --cov-report generate coverage report
```
I don't see any harm in leaving these options as default (when a developer or user run the test) that was the `runtest.py` behavior, also you could expect the same output locally and in CI